docs/monitoring-usage.rst

```diff
@@ -0,0 +1,156 @@
+Using the API
+=============
+
+
+Introduction
+------------
+
+With the Monitoring API, you can work with Stackdriver metric data
+pertaining to monitored resources in Google Cloud Platform (GCP)
+or elsewhere.
+
+Essential concepts:
+
+- Metric data is associated with a **monitored resource**. A monitored
+  resource has a *resource type* and a set of *resource labels* —
+  key-value pairs — that identify the particular resource.
+- A **metric** further identifies the particular kind of data that
+  is being collected. It has a *metric type* and a set of *metric
+  labels* that, when combined with the resource labels, identify
+  a particular time series.
+- A **time series** is a collection of data points associated with
+  points or intervals in time.
+
+Please refer to the documentation for the `Monitoring API`_ for
+more information.
+
+At present, this client library supports querying of time series,
+metric descriptors, and resource descriptors.
+
+.. _Monitoring API: https://cloud.google.com/monitoring/api/
+
+
+The Monitoring Client Object
+----------------------------
+
+The monitoring client library generally makes its
+functionality available as methods of the monitoring
+:class:`~gcloud.monitoring.client.Client` class.
```

Comment:
@rimey I'm not familiar with the `~` in Sphinx. Any link to docs? (If not, I can check out your branch and build the docs to see for myself.)